data-raw/amazon_books.R

```diff
@@ -0,0 +1,8 @@
+## code to prepare `amazon-books` dataset goes here
```

Comment:
Also, documentation for this dataset is missing.

Reply:
Can you explain how I add the documentation?